examples/flows/chat/chat-with-pdf/README.md

```diff
@@ -65,8 +65,7 @@ az configure --defaults group=<your_resource_group_name> workspace=<your_workspa
 
 ``` bash
 # create run
-pfazure run create --file batch_run.yaml --name $run_name --runtime example-runtime-ci
-# pfazure run create --file batch_run.yaml --name $run_name # automatic runtime
+pfazure run create --file batch_run.yaml --name $run_name
```

Comment:
BLOCKER: wait for the switch from client-wait to system-wait

Reply:
client wait will hit not found error with high concurrency (one requested, another one directly get)